		<launch>

			<node respawn="true" pkg="joy"
		        type="joy_node" name="joy_input" >
		    <param name="dev" type="string" value="/dev/input/js0" />
		    <param name="deadzone" value="0.12" />
		    </node>

		    <!-- Axes -->
	  		<param name="axis_x" value="0" type="int"/>
	  		<param name="axis_y" value="1" type="int"/>
	  		<param name="axis_z" value="2" type="int"/>
	 		<param name="scale_" value="10" type="double"/>
	 		<node pkg="foot_control_xy" type="joy_control" name="joyControl"/>
 		</launch>
